### PR TITLE
fix: update email sending setup

### PR DESCRIPTION
**Pull request by sbernheim4**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -78,15 +79,23 @@ app.post('/help', (req, res) =>{
...
 
-      email_sender(email)
+    // Send an email with the below
+    const options = {
+        from: 'from@from.com',
+        to: 'to@to.com',
+        subject: "You're both free now."
+    };
+
+    sendEmail(options);
```
Comment (sbernheim4) [inline src/app.js]: You'll want to wrap this in a try/catch to handle any errors that this function might throw.

```js
try {
  sendEmail(options);
} catch (error) {
  // handle error logic here 
}

**Comment by aharonhillel** [comment]
Great changes!

**Comment by aharonhillel** [comment]
e

**Comment by aharonhillel** [comment]
Merging Sam's help!

**Pull request closed without merging**